### PR TITLE
docs(css): add sizing values

### PR DESCRIPTION
**Pull request by schirrel**
Add definitions about width and height usage and to avoid magic numbers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -649,6 +649,75 @@ keep the media queries as close to their relevant rule sets whenever possible.
 }
 ```
 
+### 5.6 Spacing and size of image and components
+
+Is a commom problem to use width and height or all dynamic or all hardcoded, but each one has it own purpose. We should avoid using magic numbers at all times.
+
+> _"Magic numbers are those numbers that appear in code without explanation, but that 'magically' make things work."_  Are numbers that dont have a why, but works.
```
Comment (felipefialho) [inline README.md:656]: ❤️ ❤️ ❤️

File: README.md [hunk 2]
```diff
@@ -649,6 +649,75 @@ keep the media queries as close to their relevant rule sets whenever possible.
...
+```
+
+**❌ Bad:**
+
+```scss
+
+.logout__icon {
+    height: 25px;
+    width: 25px;
+  }
```
Comment (felipefialho) [inline README.md:684]: (lint)

```suggestion
.logout__icon {
  height: 25px;
  width: 25px;
}
```

File: README.md [hunk 3]
```diff
@@ -649,6 +649,75 @@ keep the media queries as close to their relevant rule sets whenever possible.
...
+```
+
+**❌ Bad:**
+
+```scss
+.shopfrom__banner {
+    height: calc(4 * var(--spaceing-xxxlarge);
+    width: calc(2 * var(--spacing-giant);
+  }
+}
```
Comment (felipefialho) [inline README.md:718]: (lint) and remove extra `}`
```suggestion
.shopfrom__banner {
  height: calc(4 * var(--spaceing-xxxlarge);
  width: calc(2 * var(--spacing-giant);
}
```

File: README.md [hunk 4]
```diff
@@ -649,6 +649,75 @@ keep the media queries as close to their relevant rule sets whenever possible.
...
+.shopfrom__banner {
+    height: 900px;
+    width: 480px;
+  }
+
+  @media (min-width: 991px) {
+     height: 740px;
+     width: 240px;
+  }
+}
```
[the remark this answers is not in the document]
Comment (schirrel) [reply README.md:705]: @felipefialho na verdade faltavam mais exemplos, o uso do query é pra mostrar que é errado varios valores fixos por media query

**Review by felipefialho — APPROVED** [review]

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (felipefialho) [reply README.md:705]: mas o lint segue errado, hahaha, vale ajustar 🎃

[the remark this answers is not in the document]
Comment (schirrel) [reply README.md]: @felipefialho corrigi o lint 🙏🏻

**Review by felipefialho — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -649,6 +649,86 @@ keep the media queries as close to their relevant rule sets whenever possible.
...
+### 5.6.2 Images and well defined components
+If you are using a image, or a component that has a design size and it sizes at maximum vary from desktop/mobile, use the value of it:
+
+**✅ Good:**
+
+```scss
+.shopfrom__banner {
+  height: 900px;
+  width: 480px;
+  @media (min-width: 991px) {
```
Comment (gabrielduete) [inline README.md:699]: (question): Can we add a empty space here? Similar as shown in this section of the guideline 🤔

https://github.com/juntossomosmais/frontend-guideline#55-css-media-queries

File: README.md [hunk 6]
```diff
@@ -649,6 +649,86 @@ keep the media queries as close to their relevant rule sets whenever possible.
...
+```scss
+.shopfrom__banner {
+  height: calc(4 * var(--spaceing-xxxlarge);
+  width: calc(2 * var(--spacing-giant);
+}
+// or
+.shopfrom__banner {
+  height: 480px;
+  width: 170px;
+  @media (max-width: 746px) {
```
Comment (gabrielduete) [inline README.md:717]: Same here

File: README.md [hunk 7]
```diff
@@ -649,6 +649,86 @@ keep the media queries as close to their relevant rule sets whenever possible.
...
+    height: 740px;
+    width: 240px;
+  }
+  @media (max-width: 991px) {
```
Comment (gabrielduete) [inline README.md:721]: Same here

File: README.md [hunk 8]
```diff
@@ -649,6 +649,86 @@ keep the media queries as close to their relevant rule sets whenever possible.
...
+    height: 900px;
+    width: 320px;
+  }
+  @media (max-width: 1024px) {
```
Comment (gabrielduete) [inline README.md:725]: Same here

**Review by gabrielduete — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (schirrel) [reply README.md:719]: done [b387d7c](https://github.com/juntossomosmais/frontend-guideline/pull/40/commits/b387d7c836c82d6d8e540a9d11bfaf18bb1aa117)

File: README.md (hunk 7 above)
Comment (schirrel) [reply README.md:724]: done [b387d7c](https://github.com/juntossomosmais/frontend-guideline/pull/40/commits/b387d7c836c82d6d8e540a9d11bfaf18bb1aa117)

File: README.md (hunk 8 above)
Comment (schirrel) [reply README.md:729]: done [b387d7c](https://github.com/juntossomosmais/frontend-guideline/pull/40/commits/b387d7c836c82d6d8e540a9d11bfaf18bb1aa117)

**Review by gabrielduete — APPROVED** [review]
:alanzin-jacquin:

File: README.md [hunk 9]
```diff
@@ -649,6 +649,90 @@ keep the media queries as close to their relevant rule sets whenever possible.
...
 
+### 5.6 Spacing and size of image and components
+
+Is a commom problem to use width and height or all dynamic or all hardcoded, but each one has it own purpose. We should avoid using magic numbers at all times.
+
+> _"Magic numbers are those numbers that appear in code without explanation, but that 'magically' make things work."_  Are numbers that dont have a why, but works.
+
+
+### 5.6.1 Dynamic values
+If you are using padding, margin, gap should use our [Venices's spacing variables](https://juntossomosmais.github.io/venice/vue/?path=/docs/getting-started-styles-spacings--page). Any space that override it values must be validated once our Design System is well defined around these alues and our UX Teams guide must follow it.
```
Comment (RodrigoRVSN) [inline README.md:660]:
```suggestion
If you are using padding, margin, gap should use our [Venices's spacing variables](https://juntossomosmais.github.io/venice/vue/?path=/docs/getting-started-styles-spacings--page). Any space that override it values must be validated once our Design System is well defined around these values and our UX Teams guide must follow it.
```

**Review by felipefialho — APPROVED** [review]

**Pull request merged**